script/Arcueid/Day 10/10_00_ARC10_5.txt

```diff
@@ -142,27 +152,30 @@ It's just as Arcueid said: there really isn't anyone here but us.
 [sha:a8021f316174b04a8db86aa56455fc6639e9fbeb]{
 -- Page 406, Offset 25291.
 -- アルクェイドはじっと上目遣いで見つめてくる。……なんとなく、アルクェイドの言いたい事が分かってしまった。
-Arcueid stares at me with upturned eyes... Somehow, I already knew what Arcueid wanted to say.
+// Literal 上目遣い doesn't translate super well imo,
+// compare image search results of the JP and literal EN
+Arcueid stares at me with puppy dog eyes...
```

Comment:
"Arcueid stares at me with upturned eyes..." reads perfectly naturally to me imo. With that said though, I think this version is also fine